pkg/install/v1alpha1/jiva_snapshot.go

```diff
@@ -39,7 +42,7 @@ metadata:
 spec:
   meta: |
     id: readSourceSvc
-    runNamespace: {{ .Snapshot.runNamespace }}
+    runNamespace: {{ .Config.JivaRunNamespace.value }}
```

Comment:
in the case of upgraded setup, creating snapshots of old volumes can fail as 'list'ing of jiva-svc's are done in openebs namespace here.

Reply:
what this runtask `jiva-snapshot-create-output-default` does? Does it needs any namespace?

Reply:
yes, you are right. Thanks for pointing this out.

Reply:
I'm planning to add version check and also as per discussion we won't allow creation of snapshot for older versions.